sql1.ipynb

```diff
@@ -345,58 +346,58 @@
     }
    },
    "source": [
-    "#### Load a DB object with `connect` and `cursor`"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "hidden": true
-   },
-   "outputs": [],
-   "source": [
-    "!ls"
+    "Looks like we got some data, but it's not clear what each element represents. We can view the column names in the cursor's `description` attribute."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
     "hidden": true,
+    "scrolled": false,
     "slideshow": {
      "slide_type": "fragment"
     }
    },
    "outputs": [],
    "source": [
-    "con = sqlite3.connect('flights.db')\n",
-    "cursor = con.cursor()"
+    "cursor.description"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "heading_collapsed": true,
-    "hidden": true,
-    "slideshow": {
-     "slide_type": "slide"
-    }
-   },
+   "metadata": {},
    "source": [
-    "#### Use sqlite_master to find all the tables in the schema"
+    "## `pd.read_sql()`\n",
+    "\n",
+    "We can get the data and the column names into a nice, tidy DataFrame using `pd.read_sql()`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "airports_df = pd.read_sql(airports_query, con)\n",
+    "airports_df.head()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
+    "heading_collapsed": true,
     "hidden": true,
     "slideshow": {
      "slide_type": "slide"
     }
    },
    "source": [
-    "Get the schema of a database from a db in sqlite"
+    "## Explore the Schema\n",
+    "\n",
+    "The schema of our database lives in the `sqlite_master` table. More info [here](https://www.techonthenet.com/sqlite/sys_tables/index.php)."
```

Comment:
I worry that this can be confusing since they've only seen one query so far; maybe a little more explanation that this is builtin into the database for SQLite?

Reply:
Added!